source/extensions/filters/network/zookeeper_proxy/filter.h

```diff
@@ -90,13 +146,21 @@ struct ZooKeeperProxyStats {
   ALL_ZOOKEEPER_PROXY_STATS(GENERATE_COUNTER_STRUCT)
 };
 
+enum class ErrorBudgetResponseType { FAST, SLOW, NONE };
+
+using LatencyThresholdList = const Protobuf::RepeatedPtrField<
+    envoy::extensions::filters::network::zookeeper_proxy::v3::LatencyThreshold>;
+using LatencyThreshold_Opcode =
+    envoy::extensions::filters::network::zookeeper_proxy::v3::LatencyThreshold_Opcode;
+using LatencyThreshold = envoy::extensions::filters::network::zookeeper_proxy::v3::LatencyThreshold;
```

Comment:
my bad. could we try 
`using envoy::extensions::filters::network::zookeeper_proxy::v3::LatencyThreshold_Opcode;` to remove the redundant assignment on the above two?

Reply:
I think `LatencyThreshold_Opcode` and `LatencyThreshold` are two different namespaces.
I use below in the end:
```
using envoy::extensions::filters::network::zookeeper_proxy::v3::LatencyThreshold;
using envoy::extensions::filters::network::zookeeper_proxy::v3::LatencyThreshold_Opcode;
```

Reply:
yeah you need to declare both.